### ISSUE TITLE
AzureRelay does not auto renew ssl certificate.

### ISSUE THREAD

**Issue by ahmadabdolsaheb** [AzuraCast/AzuraRelay#14]
The SSL Certificate of our relay has expired and required an `./docker.sh update-self` and `./docker.sh update` to be resolved.

I have been experiencing this issue consistently for the past year where an error with an updater would cause this issue.

I was wondering if others has come across this issue, and if there are ways we could prevent it from happening in the future.

**Comment by SlvrEagle23** [comment]
@ahmadabdolsaheb As of the latest AzuraRelay update, we've updated our infrastructure to use the same `nginx-proxy` setup that we use with the main AzuraCast docker installation format.

You may need to run `./docker.sh setup-letsencrypt` again to update the environment variables for the new setup, but after that point it should automatically manage renewal for you without needing to take care of it manually.

**Comment by ahmadabdolsaheb** [comment]
@SlvrEagle23, `setup-letsencrypt` uses `envfile-set `that does not seem to be defined in `docker.sh` for azurarelay.

I ran `./docker.sh letsencrypt-create coderadio.freecodecamp.org` on azuracast vm which seems to have set the relay’s cert to `localhost`


Additionally,  I added the `envfile-set` from azuracast `docker.sh` to azurarelay and ran `./docker.sh setup-letsencrypt`on the relay, but still no luck.

I am still debugging, but let me know if anything comes to mind.
